spec/factories.rb

```diff
@@ -52,7 +52,9 @@
     end
 
     after(:build) do |organization, evaluator|
-      create_list(:user, evaluator.users_count, organizations: [organization])
+      if evaluator.users.count < evaluator.users_count
+        create_list(:user, evaluator.users_count, organizations: [organization])
```

Comment:
Right, because a token is chosen from a random user. In that case, this makes sense to me.